doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -584,6 +584,11 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
 Simultaneous multi-image DFU
 ****************************
 
+.. note::
+
+   As described in :ref:`multi-image build guide <ug_multi_image_variables>`, make sure to the child image name prefix to the name of the Kconfig options that are used when building MCUboot as child image (for example, ``child_image_CONFIG_PCD_APP``.
```

Comment:
```suggestion
   As described in :ref:`multi-image build guide <ug_multi_image_variables>`, make sure to name the child image prefix to the name of the Kconfig options that are used when building MCUboot as a child image (for example, ``child_image_CONFIG_PCD_APP``.
```

Reply:
maybe: "to add the child image prefix (...)"?